src/components/arcgis-layer-manager/arcgis-layer-manager.js

```diff
@@ -9,28 +9,19 @@ const ArcgisLayerManager = ({ map, activeLayers, customFunctions }) => {
   const { items } = layers;
   
   useEffect(() => {
+    items.forEach(sceneLayer => {
+      const isVisible = activeLayers.some(activeLayer => activeLayer.title === sceneLayer.title)
+      if (isVisible) {
+        sceneLayer.visible = true;
+        setOpacity(sceneLayer, activeLayers);
+      } else {
+        sceneLayer.visible = false;
+      }
```

Comment:
A small suggestion here, shorter syntax:
```suggestion
      sceneLayer.visible = isVisible;
      if (isVisible) { setOpacity(sceneLayer, activeLayers); }
```